client/collections/artworks.coffee

```diff
@@ -18,6 +18,7 @@ module.exports = class Artworks extends Backbone.Collection
       new Artwork(id: id).fetch
         error: options.error
         success: (artwork) -> cb null, artwork
+        complete: options.complete
     , (err, artworks) =>
```

Comment:
The `complete` method was missing here which allows the `featureFromInput` in the mentioned/featured artworks section to be called back. Was previously holding on to the loading spinner.